src/Symfony/Component/BrowserKit/Cookie.php

```diff
@@ -126,7 +126,7 @@ public function __toString()
      *
      * @throws \InvalidArgumentException
      */
-    public static function fromString($cookie, $url = null)
+    public static function fromString(string $cookie, $url = null)
```

Comment:
Can't we add `string` to `$url` as well?

Reply:
done